Build failure: xoscope

## Steps To Reproduce

Steps to reproduce the behavior:

1. build *xoscope*

## Build Log

<details>

<summary>Build Log</summary>

```
Running phase: buildPhase
build flags: SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash
make  all-am
make[1]: Entering directory '/build/xoscope-2.3'
gcc -DHAVE_CONFIG_H -I.  -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/kv052knpj2l1zxdfxvqz0bnhbla41sxg-gtkdatabox-1.0.0/include -export-dynamic -DPACKAGE_LIBEXEC_DIR='"/nix/store/0x5819s89bzlwjiqpqdflcv6adq8m29l-xoscope-2.3/bin"' -DGTK_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE   -g -O2 -Wall -c -o xoscope.o xoscope.c
echo "/* xoscope.css generated by make from xoscope.css.in and xoscope.css.m4.*/" > xoscope.css
echo "/* DO NOT EDIT THIS FILE!                                               */" >> xoscope.css
m4 ./xoscope.css.m4 ./xoscope.css.in >> xoscope.css
sed -i -e '/^$/d' xoscope.css
echo "/* builtins.h generated by make from xoscope.css and xoscope.glade.*/" > builtins.h
echo "/* DO NOT EDIT THIS FILE!                                         */" >> builtins.h
echo "char * xoscope_css = " >> builtins.h
sed -e 's/"/\\"/g' -e 's/^/"/' -e 's/$/ "/' ./xoscope.css >> builtins.h
echo ";" >> builtins.h
echo "char * gladestring = " >> builtins.h
sed -e 's/"/\\"/g' -e 's/^/"/' -e 's/$/"/' ./xoscope.glade >> builtins.h
echo ";" >> builtins.h
echo "char * operl_program = " >> builtins.h
sed -e 's/\\/\\\\/g' -e 's/"/\\"/g' -e 's/^/"/' -e 's/$/\\n"/' ./operl.in >> builtins.h
echo ";" >> builtins.h
echo "char * operl_help_text = " >> builtins.h
sed -e 's/\\/\\\\/g' -e 's/"/\\"/g' -e 's/^/"/' -e 's/$/\\n"/' ./operl.help >> builtins.h
echo ";" >> builtins.h
gcc -DHAVE_CONFIG_H -I.  -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/kv052knpj2l1zxdfxvqz0bnhbla41sxg-gtkdatabox-1.0.0/include -export-dynamic -DPACKAGE_LIBEXEC_DIR='"/nix/store/0x5819s89bzlwjiqpqdflcv6adq8m29l-xoscope-2.3/bin"' -DGTK_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE   -g -O2 -Wall -c -o xoscope_gtk.o xoscope_gtk.c
gcc -DHAVE_CONFIG_H -I.  -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/kv052knpj2l1zxdfxvqz0bnhbla41sxg-gtkdatabox-1.0.0/include -export-dynamic -DPACKAGE_LIBEXEC_DIR='"/nix/store/0x5819s89bzlwjiqpqdflcv6adq8m29l-xoscope-2.3/bin"' -DGTK_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE   -g -O2 -Wall -c -o file.o file.c
gcc -DHAVE_CONFIG_H -I.  -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/kv052knpj2l1zxdfxvqz0bnhbla41sxg-gtkdatabox-1.0.0/include -export-dynamic -DPACKAGE_LIBEXEC_DIR='"/nix/store/0x5819s89bzlwjiqpqdflcv6adq8m29l-xoscope-2.3/bin"' -DGTK_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE   -g -O2 -Wall -c -o func.o func.c
gcc -DHAVE_CONFIG_H -I.  -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/kv052knpj2l1zxdfxvqz0bnhbla41sxg-gtkdatabox-1.0.0/include -export-dynamic -DPACKAGE_LIBEXEC_DIR='"/nix/store/0x5819s89bzlwjiqpqdflcv6adq8m29l-xoscope-2.3/bin"' -DGTK_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE   -g -O2 -Wall -c -o display.o display.c
gcc -DHAVE_CONFIG_H -I.  -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/kv052knpj2l1zxdfxvqz0bnhbla41sxg-gtkdatabox-1.0.0/include -export-dynamic -DPACKAGE_LIBEXEC_DIR='"/nix/store/0x5819s89bzlwjiqpqdflcv6adq8m29l-xoscope-2.3/bin"' -DGTK_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGSEAL_ENABLE   -g -O2 -Wall -c -o comedi.o comedi.c
comedi.c: In function 'open_comedi':
comedi.c:480:15: error: too few arguments to function 'comedi_get_cmd_generic_timed'
  480 |         ret = comedi_get_cmd_generic_timed(comedi_dev, comedi_subdevice, &cmd, 0);
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from comedi.c:27:
/nix/store/jl0birlinz2ifyqa8p6ckg1a23li3395-comedilib-0.12.0-dev/include/comedilib.h:189:5: note: declared here
  189 | int comedi_get_cmd_generic_timed(comedi_t *dev,unsigned int subdevice,
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:512: comedi.o] Error 1
make[1]: Leaving directory '/build/xoscope-2.3'
make: *** [Makefile:384: all] Error 2
error: builder for '/nix/store/hvs5icrwmflcp40q0151663ibm0kydfv-xoscope-2.3.drv' failed with exit code 2
error: 1 dependencies of derivation '/nix/store/r73h7fykb7r488ksk5l3p9q9pim4knki-man-paths.drv' failed to build
error: 1 dependencies of derivation '/nix/store/dhkikjnx2iw9xmvsfm4qibkxyx6isl7j-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/rcy17x5mvgz4c41ljm8l2dr774is4qlc-nixos-system-Bitscoper-WorkStation-25.05beta729683.88195a94f390.drv' failed to build
```

</details>

## Metadata

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.2-zen1, NixOS, 25.05 (Warbler), 25.05beta729683.88195a94f390`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.11`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

## Notify Maintainers

@infinisil @Aleksanaa 

---

Note for Maintainers: Please tag this issue in your PR.

--- a/pkgs/by-name/xo/xoscope/fix-gcc14.patch
+++ b/pkgs/by-name/xo/xoscope/fix-gcc14.patch
@@ -1,0 +1,15 @@
+--- a/configure
++++ b/configure
+@@ -7054,9 +7054,10 @@
+ 	   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
+ /* end confdefs.h.  */
+ 
++#include <stddef.h>
+ #include <comedilib.h>
+-main() {
+-       comedi_get_cmd_generic_timed(NULL, NULL, NULL, 0, 0);
++void main() {
++       comedi_get_cmd_generic_timed(NULL, 0, NULL, 0, 0);
+ }
+ _ACEOF
+ if ac_fn_c_try_compile "$LINENO"; then :

--- a/pkgs/by-name/xo/xoscope/package.nix
+++ b/pkgs/by-name/xo/xoscope/package.nix
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
     sha256 = "0a5ycfc1qdmibvagc82r2mhv2i99m6pndy5i6ixas3j2297g6pgq";
   };
 
+  patches = [ ./fix-gcc14.patch ];
+
   nativeBuildInputs = [
     pkg-config
     gnum4

